src/base/orgs/View/Projects.svelte

```diff
@@ -2,12 +2,12 @@
   import { navigate } from "svelte-routing";
   import { onMount } from "svelte";
   import type { Config } from "@app/config";
-  import * as proj from "@app/project";
+  import * as proj from "@app/base/projects/Project";
   import Loading from "@app/Loading.svelte";
   import Message from "@app/Message.svelte";
   import Widget from '@app/base/projects/Widget.svelte';
   import type { Profile } from "@app/profile";
-  import type { ProjectInfo, Anchor, PendingAnchor } from "@app/project";
+  import type { ProjectInfo, Anchor, PendingAnchor } from "@app/base/projects/Project";
```

Comment:
Personally I prefer the destructured way of importing, but I left the default `* as proj` imports. Maybe we should define when it makes sense and when not.